Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -522,6 +528,10 @@ protected override void GetChildItems(string path, bool recurse)
         /// <param name="newItemValue">The value of the item to create. We assume it is a string.</param>
         protected override void NewItem(string path, string itemTypeName, object newItemValue)
         {
+            if (!ShouldProcess(path, "New-Item"))
```

Comment:
Please add the file path here, and below in `Clear-Content` and `Remove-Item`. Using (cross-plat) PowerShell, it shows the path:

````
PS /Users/chrsmith/src/google-cloud-powershell> New-Item test.txt -WhatIf                                                                                                                                   
What if: Performing the operation "Create File" on target "Destination: /Users/chrsmith/src/google-cloud-powershell/test.txt".
````

We don't need to be as verbose, but we should at least include the file path.